inst/include/Rcpp/Dimension.h

```diff
@@ -39,14 +39,14 @@ namespace Rcpp{
 	            dims = other.dims ;
 	        return *this ;
 	    }
-	    Dimension(const size_t& n1) : dims(1){
+	    Dimension(const int& n1) : dims(1){
```

Comment:
Actually, dims is an 'int' vector so 'size_t' will be downcasted to 'int' at the end